### ISSUE TITLE
Add Ability To Remotely Start Remotely

### ISSUE THREAD

**Issue by mmccall0813** [RainwayApp/bug-tracker#314]
**Is your feature request related to a problem? Please describe.**
Yes, this is related to a problem I sometimes have. Rainway for me crashes every once in a while.

**Describe the solution you'd like**
When on the screen of "Rainway desktop not found", add a feature (that first has to be enabled on the PC's Rainway desktop settings) to attempt to start Rainway

**Describe alternatives you've considered**
The only "alternative" I can think of is just waiting until I get home and start Rainway up when I get home.

**Additional context**
Basically, when Rainway Desktop is not running and the "Rainway Desktop not found" screen is up, there should be a button that attempts to start Rainway Desktop on the last connected to computer on the account.

**Comment by healingbrew** [comment]
So basically you want the client to be able to identify offline servers, wake them up and launch Rainway as well?

**Comment by mmccall0813** [comment]
Something along those lines

**Comment by mmccall0813** [comment]
Now that I think more about this it would be really hard to implement it. Because how exactly would you contact the computer when said computer isn't running Rainway. The only thing I can think of is having some sort of background process almost always running on the computer.
